src/styles/card.css

```diff
@@ -0,0 +1,80 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
```

Comment:
card width should `200px` including border

Reply:
The card has a width of 200 pixels including the border, box-sizing in normalize.css

Reply:
ok